app/src/test/java/com/example/beatboxapp/SoundViewModelTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.beatboxapp
+
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+
+class SoundViewModelTest {
+
+    private lateinit var beatBox: BeatBox
+    private lateinit var sound: Sound
+    private lateinit var subject: SoundViewModel
+
+    @Before
+    fun setUp() {
+        beatBox = Mockito.mock(BeatBox::class.java)
+        sound = Sound("assetPath")
```

Comment:
Using a real instance of `Sound` class will also run the logic that formats the `name` parameter, it's pretty simple so it doesn't cause any problems but we don't really care about that logic here, in `SoundViewModelTest`. It should be tested in a separate test class created for the `Sound` class.
In here you could create a mock of `Sound` class and mock it's `name` parameter to return something and test that `subject.title` is the same as that string